client/defines.h

```diff
@@ -24,8 +24,11 @@
 
 typedef enum
 {
+    /* this should be a bitmask */
```

Comment:
I actually meant this for `TDNFPopulatePkgInfoArray()` to fill in details for bits that are set in a bitmask. Not for PTDNF_REPOQUERY_ARGS.

This should. be another change since it affects the `list` and `info` commands as well.